src/Components/Seo/ArtistStructuredData.tsx

```diff
@@ -0,0 +1,126 @@
+import { StructuredData } from "Components/Seo/StructuredData"
+import { getENV } from "Utils/getENV"
+import type { ArtistStructuredData_artist$key } from "__generated__/ArtistStructuredData_artist.graphql"
+import compact from "lodash/compact"
+import { graphql, useFragment } from "react-relay"
+import type { Thing, WithContext } from "schema-dts"
+
+interface Props {
+  artist: ArtistStructuredData_artist$key
+}
+
+export const ArtistStructuredData: React.FC<Props> = ({ artist }) => {
+  const data = useFragment(fragment, artist)
+  const artistUrl = `${getENV("APP_URL")}${data.href}`
+
+  const creatorOf = data.artworks_connection?.edges
+    ?.map(edge => {
+      const artwork = edge?.node
+      if (!artwork || !artwork.title || !artwork.href) return null
+      const artworkUrl = `${getENV("APP_URL")}${artwork.href}`
+
+      return {
+        "@type": "VisualArtwork",
+        name: artwork.title,
+        dateCreated: artwork.date,
+        artform: artwork.category,
+        url: artworkUrl,
+        image: artwork.image?.large,
+      }
+    })
+    .filter(Boolean)
+
+  const memberOf = compact(
+    data.artworks_connection?.edges?.map(edge => {
+      const partner = edge?.node?.partner
+      return partner?.name && partner?.href
+        ? {
+            "@type": "ArtGallery",
+            name: partner.name,
+            url: `${getENV("APP_URL")}${partner.href}`,
+          }
+        : null
+    }),
+  )
+
+  return (
+    <StructuredData
+      schemaData={
+        {
+          "@context": "https://schema.org",
+          "@graph": [
+            {
+              "@type": "Person",
+              "@id": artistUrl,
+              additionalType: "Artist",
+              birthDate: data.birthday,
+              deathDate: data.deathday,
+              description: data.meta?.description,
+              gender: data.gender,
+              image: data.coverArtwork?.image?.large,
+              mainEntityOfPage: artistUrl,
+              name: data.name,
+              url: artistUrl,
+              nationality: data.nationality
+                ? { "@type": "Country", name: data.nationality }
+                : undefined,
+              creatorOf: creatorOf?.length ? creatorOf : undefined,
+              memberOf: memberOf.length ? memberOf : undefined,
+            },
+            {
+              "@type": "WebPage",
+              "@id": artistUrl,
+              name: data.meta?.title,
+              description: data.meta?.description,
+              mainEntity: {
+                "@id": artistUrl,
+              },
+            },
+          ],
+        } satisfies WithContext<Thing>
```

Comment:
I'm still getting a similar type error because of `@graph` even when just importing graph from 'schema-dts' and I got caught in a circle trying to address this.  @dzucconi 
![Screenshot 2025-05-19 at 5 08 52 PM](https://github.com/user-attachments/assets/03febd6d-1d05-4e7c-80c6-e1ffd13b1095)